Content.Server/Lathe/LatheSystem.cs

```diff
@@ -127,10 +127,17 @@ private void OnInteractUsing(EntityUid uid, LatheComponent component, InteractUs
             var lastMat = string.Empty;
             foreach (var mat in material.MaterialIds)
             {
-                storage.InsertMaterial(mat, component.VolumePerSheet * multiplier);
+                storage.InsertMaterial(mat, material.volume * multiplier);
                 lastMat = mat;
             }
-            /// We need the prototype to get the color
+            
+            // Play a sound when inserting, if any
+            if (component.InsertingSound != null)
+            {
+                SoundSystem.Play(Filter.Pvs(component.Owner), component.InsertingSound.GetSound(), component.Owner);
```

Comment:
Pass in EntityManager to the filter.

Reply:
Can you elaborate? I basically copied what Rane did with the producing sound and just added it into the inserting function

Reply:
Solved